contrib/babelfishpg_common/sql/upgrades/other_types--3.2.0--3.3.0.sql

```diff
@@ -0,0 +1,3 @@
+------------------------------------------------------------------------------
```

Comment:
1. Please rename the file to babelfish_common_helper--3.2.0--3.3.sql
2. Move the existing content of babelfish_common--3.2.0-3.3.sql file to this file and remove the existing babelfish_common--3.2.0-3.3.sql file.

Reply:
1. I have renamed the file
2. it's first time we are generating babelfish_common--3.2.0-3.3.0.sql file so there's no pre existing babelfish_common--3.2.0-3.3.0.sql file